standard/expressions.md

```diff
@@ -1459,6 +1492,45 @@ The run-time processing of a delegate invocation of the form `D(A)`, where `D`
 
 See [§20.6](delegates.md#206-delegate-invocation) for details of multiple invocation lists without parameters.
 
+### §null-conditional-invocation-expression Null Conditional Invocation Expression
```

Comment:
Is this actually implemented? I've never seen it before. Is the expectation that it should make something like the code below compile?

```csharp
Action action = null;
action?();
```

That *doesn't* compile, whereas `action?.Invoke();` does.

Perhaps this was a part of the feature proposal that was dropped but not removed from the text?

Reply:
Note: it's possible that we need a *null_conditional_invocation_expression* as well as *null_conditional_member_access* in order for `action?.Invoke()` (or any other sort of invocation via a null conditional operator) to work - I'm not immediately clear whether that counts as a *null_conditional_member_access* where the result is a method group that is then invoked. (Brain not quite working right now.)

Reply:
I didn't originally include as I was using MS's C#6 Language Reference as the informal spec to work form, then I noticed it in PR #7 so added it assuming I'd missed it elsewhere. Following your query I did I wider search and found a reference on SO that it doesn't exist due to too many parsing issues. I'll remove it.